FirebaseFunctions/Tests/ObjCIntegration/FIRFunctions+Internal.h

```diff
@@ -16,7 +16,7 @@
 
 // This file started as a copy of FirebaseFunctions/Sources/FIRFunctions+Internal.h with only the
```

Comment:
Should this comment now be updated that the file doesn't exist?

Reply:
Yep. Removed, since this will now be the only variation.